### ISSUE TITLE
Grammar accepted on playground is rejected by "peglint"

### ISSUE THREAD

**Issue by mingodad** [yhirose/cpp-peglib#215]
With the grammars attached I can use then with `packrat` on the playground but on command line `peglint` only the `kotlin.peglib` works the `kotlin-perf.pegli` gives this error:
```
../../cpp-peglib0/build/lint/peglint --packrat --ast kotlin-perf.chpeg kotlin.kt
kotlin-perf.chpeg:9:5: syntax error
Command exited with non-zero status 255
```
[kotlin-peglib.zip](https://github.com/yhirose/cpp-peglib/files/8938098/kotlin-peglib.zip)

**Comment by yhirose** [comment]
@mingodad, I can't reproduce the problem...

```
> peglint --packrat --ast kotlin-perf.peglib kotlin.kt > /dev/null
>
```

**Comment by mingodad** [comment]
Thank you !
After I received the message of closing this issue I went back and tried to reproduce the problem and at first I was getting the error, then I did a copy and paste from your reply and it worked. After several experiments I found my mistake, I was using the wrong grammar as shown on my first message `kotlin-perf.chpeg` I didn't noticed that I was using the wrong extension `.chpeg` instead of `.peglib`.
Again sorry by the noise and thank you again for your help and great work.
